package.json

```diff
@@ -10,7 +10,8 @@
   },
   "devDependencies": {
     "eslint": "3",
-    "mocha": "3"
+    "mocha": "3",
+    "eslint-plugin-mocha": "4.9.0"
```

Comment:
I *think* that with this just being a dev dependency, it won't actually be included when other repos use eslint-plugin-fxa. A way to be sure is if you adjust the dependency in the fxa-auth-server to point at this branch, npm update, add `only` to one of the tests, and then run `grunt eslint`, it should fail the lint.